macros/WebGLSidebar.ejs

```diff
@@ -112,59 +112,75 @@ var text = mdn.localStringMap({
 <section id="Quick_Links">
  <ol>
   <li><a href="/<%=locale%>/docs/Web/API/WebGL_API"><strong><%=text['WebGL_API']%></strong></a></li>
-  <li data-default-state="open"><a href="/<%=locale%>/docs/Web/API/WebGL_API/Tutorial"><strong><%=text['WebGL_tutorial']%></strong></a>
-   <ol>
-     <li><a href="/<%=locale%>/docs/Web/API/WebGL_API/Tutorial/Getting_started_with_WebGL"><%=text['Getting_started']%></a></li>
-     <li><a href="/<%=locale%>/docs/Web/API/WebGL_API/Tutorial/Adding_2D_content_to_a_WebGL_context"><%=text['2D_content']%></a></li>
-     <li><a href="/<%=locale%>/docs/Web/API/WebGL_API/Tutorial/Using_shaders_to_apply_color_in_WebGL"><%=text['shaders_color']%></a></li>
-     <li><a href="/<%=locale%>/docs/Web/API/WebGL_API/Tutorial/Animating_objects_with_WebGL"><%=text['Animating']%></a></li>
-     <li><a href="/<%=locale%>/docs/Web/API/WebGL_API/Tutorial/Creating_3D_objects_using_WebGL"><%=text['3D_objects']%></a></li>
-     <li><a href="/<%=locale%>/docs/Web/API/WebGL_API/Tutorial/Using_textures_in_WebGL"><%=text['Textures']%></a></li>
-     <li><a href="/<%=locale%>/docs/Web/API/WebGL_API/Tutorial/Lighting_in_WebGL"><%=text['Lightning']%></a></li>
-     <li><a href="/<%=locale%>/docs/Web/API/WebGL_API/Tutorial/Animating_textures_in_WebGL"><%=text['Animating_textures']%></a></li>
-   </ol>
+  <li class="toggle">
```

Comment:
The ``{{WebGLSidebar}}`` is converted correctly 👍, as seen on https://developer.mozilla.org/en-US/docs/Web/API/WebGL_API/Tutorial/Getting_started_with_WebGL

<img width="469" alt="pr 789 webglsidebar" src="https://user-images.githubusercontent.com/286017/45395582-6ecb3f80-b5fc-11e8-822d-0f229c62cada.png">